health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,333 @@
+      {
+        name: "Light Theme",
+        url: "https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png",
+        minZoom: 1,
+        maxZoom: 20,
+        attribution:
+          "&copy; <a href='https://carto.com/'>CARTO</a>, " +
+          "&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors",
+      },
+    ];
+    if (!BaseMapLayers || (BaseMapLayers && BaseMapLayers.length === 0)) return;
+    let baseMaps = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'baseMaps' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)